docs/sources/setup-grafana/installation/_index.md

```diff
@@ -24,7 +24,7 @@ Grafana relies on other open source software to operate. For a list of open sour
 Grafana supports the following operating systems:
 
 - [Debian/Ubuntu]({{< relref "debian/" >}})
-- [Red Hat/RHEL/Fedora]({{< relref "redhat-rhel-fedora/" >}})
+- [Red Hat/RHEL/Fedora]({{< relref "rpm/" >}})
```

Comment:
It is important to prefix the link with either the current working directory (`.`), the parent directory (`..`), or the site root (`/`) to make sure that the link doesn't resolve ambiguously when two pages are called `rpm`. If it resolves ambiguously it 404s because Hugo doesn't know which one to pick of the multiple destinations.

- https://grafana.com/docs/writers-toolkit/review/doc-validator/errata/#relref-can-resolve-ambiguously

It's convenient to avoid the trailing slash because that means the link won't break if the destination file is changed between being a page (`page.md`), leaf bundle (`page/index.md`), or a branch bundle (`page/_index.md`).

- https://grafana.com/docs/writers-toolkit/review/doc-validator/errata/#relref-has-trailing-slash



```suggestion
- [Red Hat/RHEL/Fedora]({{< relref "./rpm" >}})
```